internal/controller/numaresourcesscheduler_controller_test.go

```diff
@@ -39,15 +34,19 @@ import (
 
 	depmanifests "github.com/k8stopologyawareschedwg/deployer/pkg/manifests"
 	depobjupdate "github.com/k8stopologyawareschedwg/deployer/pkg/objectupdate"
+	"github.com/onsi/ginkgo/v2"
```

Comment:
do we want testing frameworks imports grouped with other packages?

Reply:
it's considered as a third party so it's fine